woocommerce/woocommerce-ios
# [Mobile Payments] Add feature flag and Tap to Pay on iPhone row for TTP M2

## Description


In M2 of Tap to Pay on iPhone, we will add a set-up and test payment flow, accessible from the Payments menu, to help merchants set up and understand TTP before they have a customer with them.

This PR adds a feature flag for M2, and a row to settings for this new flow. The row does not open anything yet.

## Testing instructions


Open `Menu > Payments` using a local build from Xcode.

Observe that there is a new Tap to Pay on iPhone row present, in its own section, if you have the experimental feature toggle for `Tap to Pay on iPhone` on. (This PR does not add a new toggle)

* If you don't have the experimental feature toggle on, the new row is not shown.
* If you use a non-US store, it's should not be shown.

N.B. I also fixed a minor cell reuse issue which I noticed: sometimes, if you had the Manage Card Reader cell disabled because onboarding wasn't complete, other cells would be shown as disabled due to cell reuse. I don't have any 100% repro steps for this, but going back and forth to the screen used to trigger it.

## Screenshots

![New TTP row shown on Payments menu](https://user-images.githubusercontent.com/2472348/221247102-16520966-7868-4d5e-84b4-8dadaddd7286.jpg)


---
